R/derive_blfl.R

```diff
@@ -51,36 +38,9 @@ dtc_datepart <- function(dtc, partial_as_na = TRUE) {
 #'   seconds should be ignored (default is `TRUE`).
 #'
 #' @return Character vector containing ISO 8601 times.
-#' @export
```

Comment:
Did we decide to not export this anymore?

Reply:
@edgar-manukyan thanks for checking! This function contains both "@keywords internal" and "@export". As I was trying to replace all the "sdtm:::" from the package, I included this one as well. Also, since `dtc_timepart()` is an internal function, I thought this might be the same. @ramiromagno could you confirm?

Reply:
Hi @ShiyuC and @edgar-manukyan : these functions are part of a PR (see https://github.com/pharmaverse/sdtm.oak/commit/6abb5bac6a10fd6cfeaa936b7623d9feaa9aea49) by @kamilsi and co-authored by @rammprasad and @edgar-manukyan :) I'm not sure what the original intention was...

Reply:
But I am guessing the idea was to have these functions as internal only.

Reply:
Confirmed with Ram that this function should be internal